regions_www/tabs/manual_body.html

```diff
@@ -23,6 +23,10 @@ <h4><a name="essentials"></a>PCT Essentials</h4>
           <ul style="line-height:2.2;">
           <li>Read our two pager <a target="_blank" href="https://cdn.rawgit.com/npct/pct-shiny/30b98621b37da3cb66fcf2253583af9f026f9fce/regions_www/www/static/03d_other_reports/2020-pct-essentials.pdf">Essentials</a> document first, for a high level summary</li>
           </ul>
+          <h4><a name="advanced-training"></a>Advanced training guide (for developers / R users)</h4>
```

Comment:
This is missing the link on top

```html
<div style="padding-bottom:10px;"><a class="manuallinks" href="#advanced-training">Advanced training guide</a></div>
 ```

Reply:
Thanks Nikolai. Updated as suggested, although not 100% sure this is now right.